packages/atomic/src/components/search/result-lists/atomic-folded-result-list/e2e/atomic-folded-result-list.e2e.ts

```diff
@@ -1,10 +1,94 @@
 import {test, expect} from './fixture';
 
-// TODO: KIT-3546 - Make this test pass
-test.describe('When no child results', () => {
-  test.fixme('should show a "no results" label', async ({foldedResultList}) => {
+test.describe('when more results are NOT available & there are NO result children', () => {
```

Comment:
No need to shout at us 😄 
```suggestion
test.describe('when more results are not available & there are no result children', () => {
```

Reply:
This is intended, imo without it the test names are too similar and it is confusing.